src/NHibernate/Context/WcfOperationSessionContext.cs

```diff
@@ -1,3 +1,5 @@
+#if !NETSTANDARD2_0
```

Comment:
This is supported through [System.ServiceModel.Primitives](https://www.nuget.org/packages/System.ServiceModel.Primitives) (.NetStandard + Platform Extensions)